conf/app.routes

```diff
@@ -4,9 +4,9 @@
 
 GET         /dummy                                       controllers.DummyController.onPageLoad
 
-GET         /categorisation-start                        controllers.CategoryGuidanceController.onPageLoad
+GET         /category-guidance                           controllers.CategoryGuidanceController.onPageLoad
```

Comment:
should be categorisation-start as per confluence: https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TGP&title=Categorisation+information